docs/pages/contributing/documentation/style-guide.mdx

```diff
@@ -1,9 +1,29 @@
 ---
-title: Style Guide
-description: Use this reference to ensure our docs are consistent and effective
+title: Style guide
+description: Provides guidelines, documentation conventions, and recommended practices for contributing to Teleport documentation.
+keywords:
+  - documentation
+  - contributors
+  - community
 ---
 
-To keep our documentation effective as Teleport develops, we need to ensure that we maintain a consistent approach to organizing information.
+This guide describes our approach to building effective documentation, 
+the conventions we follow, and the recommended practices for organizing, crafting, 
+and evolving our information architecture with a clear and consistent voice.
+
+Many style rules have equally justifiable alternatives. Following a common style guide 
+is intended to prevent debates over style questions from getting in the way of 
+documenting Teleport. If you don't see an answer to a style question in this guide, 
+follow your best judgement and keep the style you choose consistent within the 
+page you're working on.
+
+If you're looking for additional resources, the following guides are the most frequently
+used:
+
+- [Google developer documentation style guide](https://developers.google.com/style)
+- [Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/welcome/)
+- [Apple Style Guide](https://help.apple.com/pdf/applestyleguide/en_US/apple-style-guide.pdf)
+- [Chicago Manual of Style](http://www.chicagomanualofstyle.org/)
```

Comment:
I would remove this list, since much of this material consists of decisions among equally justifiable style alternatives, rather than guidance for technical writing. We also can't guarantee that these style guides are consistent with the Teleport style guide.